scripts/exregional_plot_allvars.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+#pylint: disable=all
```

Comment:
It's that bad, eh?

Reply:
Oh yeah. I don't even want to touch this thing!